content/pt-br/container-orchestration.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Orquestração de Contêiners
+status: Completed
+category: conceito
+---
+
+
+Orquestração de [Contêiners](/container/) refere-se à gestão e automatização do ciclo de vida de aplicativos [contêinerizados](/containerization/) em ambientes dinâmicos. 
+É executado através de um Orquestrador de Contêiners ([Kubernetes](/kubernetes) na maioria dos casos). Plataformas que possibilitam e facilitam a implantação controlada de multiplos aplicativos,  o seu [escalonamento](/scalability/) automatizado, a recuperação automática em caso de falha e monitoramento.
```

Comment:
The links need to be updated to point to the pt-br version, like:
* pt-br/container
* pt-br/containerization
* pt-br/kubernetes
* pt-br/scalability

Please take a look and update the other occurrences.